collections/georgiaStateUsers.js

```diff
@@ -0,0 +1,95 @@
+var o = {optional: true};
+
+BrandedUserSchema = new SimpleSchema({
+	"_id": {type: String, regEx: SimpleSchema.RegEx.Id, optional: true},
+	crmId: fields.string({optional: false}), //gsu unique id
+	created: fields.date(),
+	enrollmentId: fields.string(o), //pantherId for gsu
+	schoolEmail: fields.email(o),
+	entryYear: fields.number(o),
+	entryTerm: fields.string(o),
+	name: {type: new SimpleSchema({
+		first: fields.string(o),
+		last: fields.string(o),
+		middleInitial: fields.string(o),
+		full: fields.string(o),
+		nickName: fields.string(o),
+	}),optional: true},
+	phone: fields.phone_number(o),
+	dob: fields.date(o),
+	location: {type: new SimpleSchema({
+		address1: fields.address(o),
+		address2: fields.address(o),
+		city: fields.string(o),
+		state: fields.state(o),
+		zip: fields.zip_code(_.extend(o)),
+		country: fields.string(o),
+	}), optional: true},
+	inStateStudent: fields.bool(o),
+	application: {type: new SimpleSchema({
+		status: fields.string(o), 
+	}), optional: true},
+	profile: {type: new SimpleSchema({
+		studentType: fields.string(o),
+	}), optional: true},
+	finAid: {type: new SimpleSchema({
+		fafsaComplete: fields.bool(o),
+		finAidInterest: fields.bool(o),
+		scholarshipAwarded: fields.bool(o),
+		scholarshipAccepted: fields.bool(o),
+		missingEntryLoan: fields.bool(o),
+		missingPromissoryLoan: fields.bool(o),
+	}),optional: true},
+	housing: {type: new SimpleSchema({
+		onCampus: fields.bool(o),
+		preferenceType: fields.string(o),
+		depositPaid: fields.bool(o),
+		depositDate: fields.bool(o),
+	}),optional: true},
+	orientation:{type: new SimpleSchema ({
+		needsToRsvp: fields.bool(o),
+		attended: fields.bool(o),
+		attendedDate: fields.date(o),
+		registeredDate: fields.date(o),
+	}),optional: true},
+	textSetting:{type: new SimpleSchema ({
```

Comment:
@AndrewMagliozzi Good point. "softStops" are currently handled by `stopBot` (and sets `pausedUntil`) - that may want to be rethought for branded interactions.

I had forgot that different hard stops can be triggered for the different messagingServices. We'll probably want to update our schema to reflect that, and update our [`handleHardStopStart`](handleHardStopStart) and [`stopUnlessCanText `](https://github.com/AdmitHub/oli/blob/master/lib/transports/twilio.js#L416-L430) middlewares.

Nearly all of the middleware stack uses the `user` object (which is parsed out in [this middleware](https://github.com/AdmitHub/oli/blob/master/lib/transports/twilio.js#L162-L226)). Without using a user object, you'll need to refactor quite a bit of code, and then maintain those two branches moving forward. It may be simpler to have a minimal user object for these branded users that only holds their phone number, hard/soft stop states, and workflow states.

Enrollment emails are sent very deliberately within the Oli workflows, and can be avoided in the branded workflows.